docs/performance/tune-performance.md

```diff
@@ -24,9 +24,9 @@ This document covers basic tools and knobs that can be leveraged to find the bes
 ### ONNX GO Live Tool
 {: .no_toc }
 
-The [ONNX Go Live "OLive" tool](https://github.com/microsoft/OLive) is an easy-to-use pipeline for converting models to ONNX and optimizing performance with ONNX Runtime. The tool can help identify the optimal runtime configuration to get the best performance on the target hardware for the model.
+The [ONNX Go Live "OLive" tool](https://github.com/microsoft/OLive) is a python package that automates the process of accelerating models with ONNX Runtime(ORT). It contains two parts including model conversion to ONNX with correctness checking and auto performance tuning with ORT. Users can run these two together through a single pipeline or run them independently as needed.
```

Comment:
Please capitalize Python. Instead of "two parts including...", suggest "two parts: (1)... (2)..." since this is clearer on the 2 functional goals for the package. Is there a need to link to the package directly, or should people always refer to the tutorials first?

Reply:
Thanks. I have modified the description. 
User can refer to the package link first and decide to run optimization service through command line or notebooks. They can go to tutorials directly as well.